src/Models/BulkDragBoundaryConditions.jl

```diff
@@ -0,0 +1,508 @@
+"""
+    BulkDragBoundaryConditions
+
+Module for implementing quadratic drag boundary conditions for velocity fields on
+all domain boundaries (west, east, south, north, bottom, top) and immersed boundaries.
+
+Provides `BulkDragFunction` for computing momentum fluxes using bulk aerodynamic
+formulas. The drag function computes a quadratic drag on the tangential velocity:
+
+```math
+τᵘ = - Cᴰ |U| u
+```
+
+where `Cᴰ` is the drag coefficient and `|U| = √(u² + v² + w²)` is the 3D speed.
```

Comment:
```suggestion
Provides `BulkDragFunction` for computing momentum fluxes using bulk aerodynamic
formulas. The drag function computes a drag on the tangential velocity. For example for
a quadratic drag:

```math
τᵘ = - Cᴰ |U| u
```
where `Cᴰ` is the drag coefficient and `|U| = √(u² + v² + w²)` is the 3D speed.
```